src/themes/default/modules/dropdown.overrides

```diff
@@ -35,25 +35,25 @@
 
 .ui.dropdown .item .left.dropdown.icon::before,
 .ui.dropdown .left.menu .item .dropdown.icon::before {
-    content: "\f0d9"/*rtl:"\f0da"*/;
+    content: '\f0d9'/*rtl:'\f0da'*/;
```

Comment:
if https://github.com/fomantic/Fomantic-UI/pull/2594/files/ebc0cf12f5c9a7644db1757fa0815a262a4821bf#r1048963174 is parsed by the build script somehow, please test if single quotes works, both should work